src/ThriftSQL/Query.php

```diff
@@ -32,4 +32,9 @@ public function fetch( $numRows );
    * @return self
    */
   public function close();
+
+  /**
+   * @return array
+   */
+  public function schema();
```

Comment:
Maybe call this `getSchema()` or add some other verb to make it clear what this function does.

Reply:
I just referred to `PDO`'s naming style, eg: PDO::lastInsertId, PDO::errorInfo.

IMO `getXxx` usually means the an object's property, but `schema` belongs to the response, it's not a real object's property.